.github/workflows/backend-test.yml

```diff
@@ -0,0 +1,27 @@
+name: Backend Test
+
+on:
+  pull_request:
+    paths:
+      - telos-backend/**
+  push:
+    paths:
+      - telos-backend/**
+
+defaults:
+  run:
+    working-directory: telos-backend
+
+jobs:
+  test:
+    name: Running Backend Tests
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
```

Comment:
Why does this use matrix.node-version but the lint files define the version number?

Reply:
Forgot to change this after pasting from a sample 😦 
Changed to use Node.js v14